src/OpenSpaceNetArgs.cpp

```diff
@@ -439,183 +438,166 @@ void OpenSpaceNetArgs::printUsage(Action action) const
     cout << endl;
 }
 
+enum ArgUse {
```

Comment:
The problem with that is that these are all dependent on other arguments.  E.g. setting `--image` means `--map-id` is ignored.  That isn't known a-priori or causally.